make/buf/scripts/release.bash

```diff
@@ -27,6 +27,7 @@ goarch() {
     x86_64) echo amd64 ;;
     arm64) echo arm64 ;;
     aarch64) echo arm64 ;;
+    armv7l) echo arm ;;
```

Comment:
```suggestion
    armv7) echo arm ;;
```

We want to build for `armv7`, which would incorporate `armv7l` (since the architecture we're building for is `armv7`).

Reply:
Yes, makes sense. My mistake.